os/arch/arm/src/amebasmart/amebasmart_serial.c

```diff
@@ -1213,14 +1274,78 @@ static uint32_t rtk_uart_resume(uint32_t expected_idle_time, void *param)
 {
 	(void)expected_idle_time;
 	(void)param;
+
+	ALIGNMTO(CACHE_LINE_SIZE) u8 flag[CACHE_LINE_ALIGMENT(64)];
+	ALIGNMTO(CACHE_LINE_SIZE) u32 uart_data[16];
+
+	/* reset buffer and head pointer for FIFO drain buffer */
+	g_uart1_dataleft = 0;
+	memset(g_uart1_buf, 0, sizeof(g_uart1_buf));
+	g_uart1_buf_head = &g_uart1_buf;
+
+	IPC_MSG_STRUCT ipc_req_msg  __attribute__((aligned(64)));
+	ipc_req_msg.msg_type = IPC_USER_POINT;
+	ipc_req_msg.msg = (u32)uart_data;
+	ipc_req_msg.msg_len = sizeof(uart_data);
+	ipc_req_msg.rsvd = (u32)flag;
+
+	memset(flag, 0, sizeof(flag));
+	memset(uart_data, 0, sizeof(uart_data));
```

Comment:
uart_data type is u32, but memset is working in byte (u8). 
Could you compare `for` loop for each member set with 0 to memset?